README.md

```diff
@@ -36,10 +36,17 @@ You can also manually test the pipeline task by using the Azure DevOps project <
 This project contains a pipeline with our build task.
 Configure this task for your manual test and run it.
 
-- Run `npm run package` to create a .vsix package
-- Go to <https://cqse.visualstudio.com>
-- Go to _Organization Settings > Extensions_
-- Uninstall the extension and then install your package
+- Change the name of the extension in ./vss-extension.json to `teamscale-azure-devops-plugin-test`.
+- Change the `public` to `false` in ./vss-extension.json.
+- Make sure that the version number in ./vss-extension.json is larger than the one used on the Marketplace
+- Change the ID of the pipeline task in ./reportUploadTask/task.json to `8e8b64dc-cc55-4bd1-85da-7b5fde8efea2`.
+- Run `npm run package` to create a .vsix package.
+- Go to <https://marketplace.visualstudio.com/manage/publishers/cqsegmbh>.
+- Right-click the **private** version of the Teamscale DevOps extension.
+- Select _update_.
+- Upload your .vsix package.
+- Right-click the private version of the Teamscale DevOps extension and go to the Marketplace and install it.
```

Comment:
When the private extension is uploaded for the first time, it has to be shared with the organization before it can be installed: right-click the extension, click share and enter "cqse". Currently, we have the public (and unpublished) test extension which cannot be updated using a private one (and cannot be deleted for some reason 🤔). So either stick to the "old" process (public extension and unpublish) or mention that the private extension must be shared if not already done?

Reply:
thanks, added the sharing step to the instructions :-)